Library/Homebrew/cask/audit.rb

```diff
@@ -528,13 +528,13 @@ def audit_livecheck_min_os
       return unless cask.livecheckable?
       return if cask.livecheck.strategy != :sparkle
 
-      out, _, status = curl_output("--fail", "--silent", "--location", cask.livecheck.url)
-      return unless status.success?
+      result = curl_output("--fail", "--silent", "--location", cask.livecheck.url)
+      return unless result.status.success?
```

Comment:
@MikeMcQuaid Wouldn't it be better to just use `result.success?` here and in other similar places?

Reply:
@Kentzo Sorry, I'm not sure I understand why the code here needs to be changed when it's nothing to do with the `sudo` changes in your PR?